include/sdf/parser.hh

```diff
@@ -417,6 +417,15 @@ namespace sdf
   SDFORMAT_VISIBLE
   bool recursiveSiblingUniqueNames(sdf::ElementPtr _elem);
 
+  /// \brief Check that all sibling elements do not the delimiter double colons
```

Comment:
missing a verb here: "Check that all sibling elements do not the delimiter double colons"

do not *contain*?

Reply:
Yup, thanks! ac6ec01a8bfde7cc3e36fdac8c3f3d6b6ec4ac52